src/kedro_templar/core/utils.py

```diff
@@ -0,0 +1,68 @@
+import yaml
+import s3fs
+import logging
+import os
+from pathlib import Path
+from typing import Text, Dict
+from ..import settings
+
+log = logging.getLogger(__name__)
+
+
+def check_folder_structure():
```

Comment:
Are we sure we need this? This doesn't work correctly if user provided other paths using command arguments.